torch/distributions/transformed_distribution.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, base_distribution, transforms):
         event_dim = max([len(self.base_dist.event_shape)] + [t.event_dim for t in self.transforms])
         batch_shape = shape[:len(shape) - event_dim]
         event_shape = shape[len(shape) - event_dim:]
-        super(TransformedDistribution, self).__init__(batch_shape, event_shape)
+        super(TransformedDistribution, self).__init__(batch_shape, event_shape, validate_args=validate_args)
 
     @constraints.dependent_property
```

Comment:
On second thought, this just seems wrong. I think this should be
```py
params = constraints.dependent
```

Reply:
can u be more specific? Where exactly should i add this? Should i remove the annotations ?

Reply:
I believe this is simply a bug in `TransformedDistribution` which you can fix by changing just this file:
```diff
- @constraints.dependent_property
- def params(self):
-     return self.base_dist.params
+ params = constraints.dependent
```

Reply:
Done. Also fixed one test. Let me know if this looks good enough for merge.